asset/script/main.js

```diff
@@ -151,6 +153,17 @@
         });
     };
 
+    function summaryOutputControl(element) {
+        $(document).on('lint-results', function (event, results) {
+            var tooHigh = results.readingLevel >= 9;
+            console.log(tooHigh);
+            element.innerHTML = templates.readingLevel.render(
+                {"readingLevel": results.readingLevel,
```

Comment:
It's a little cleaner to create the object beforehand and just pass the object to the render function.